docs/_packages/oauth.md

```diff
@@ -135,13 +135,15 @@ If you decide you need custom success or failure behaviors (ex: wanting to show
 
 ```javascript
 const callbackOptions = {
-  success: (installation, metadata, req, res) => {
+  success: (installation, installOptions, req, res) => {
```

Comment:
installOptions or installerOptions? Since we refer to installerOptions elsewhere, perhaps we make it consistent here?

Reply:
It looks like the [documentation always uses `installOptions`](https://slack.dev/node-slack-sdk/oauth). I mistyped `installerOptions` in the pull request description, but that was my own mistake (I always want to call it `installerOptions` for some reason...).

@srajiang how do you feel about keeping it consistent with `installOptions`?

Reply:
Ah interesting `installerOptions` is a Bolt-JS specific term. Sounds good to keep it consistent as `installOptions`! Thanks for double-checking